Source/Scene/WebMapTileServiceImageryProvider.js

```diff
@@ -49,6 +55,9 @@ define([
      * @param {String} options.style The style name for WMTS requests.
      * @param {String} options.tileMatrixSetID The identifier of the TileMatrixSet to use for WMTS requests.
      * @param {Array} [options.tileMatrixLabels] A list of identifiers in the TileMatrix to use for WMTS requests, one per TileMatrix level.
+     * @param {Clock} [options.clock] A Clock instance that is used when determining the value for the time dimension. Required when options.times is specified.
```

Comment:
Nowhere else in Cesium to we link to a Sandcastle example in the jsdoc as far as I can tell. Do we want to link to the public cesiumjs.org Sandcastle or to a localhost link?